ksp_plugin/renderer.cpp

```diff
@@ -265,6 +266,20 @@ RigidTransformation<World, Navigation> Renderer::WorldToPlotting(
          WorldToBarycentric(time, sun_world_position, planetarium_rotation);
 }
 
+Rotation<CameraReference, World> Renderer::CameraReferenceRotation(
+    Instant const& time,
+    Rotation<Barycentric, AliceSun> const& planetarium_rotation) const {
+  Permutation<CameraReference, Navigation> const celestial_mirror(
+      Permutation<CameraReference, Navigation>::XZY);
+  auto const result =
+      OrthogonalMap<WorldSun, World>::Identity() *
```

Comment:
Use `Identity` instead of `OrthogonalMap`, we might be able to do better checks that way in the future.

Reply:
But then I need to `Forget` the identity for composition anyway; it should not be harder to put a static_assert or SFINAE on `OrthogonalMap<F, T>::Identity` than on `Identity<F, T>`.